arangod/Transaction/Methods.h

```diff
@@ -206,20 +206,20 @@ class Methods {
   Result begin();
 
   /// @deprecated use async variant
-  auto commit() noexcept -> Result;
+  [[nodiscard]] auto commit() noexcept -> Result;
```

Comment:
Maybe also add deprecated?
```suggestion
  [[nodiscard, deprecated("use async variant")]] auto commit() noexcept -> Result;
```